### DISCUSSION TITLE
[프론트엔드] 함수형 프로젝트를 클론하여 실행 시 에러 발생.

### DISCUSSION THREAD

**Discussion by cocoburger** [fsoftwareengineer/todo-application#114] [질문과 답변]
## 1. [object Object] 가 기본으로 input창 value 값으로 들어가 있음.
![image](https://user-images.githubusercontent.com/73986381/166086475-8dea2aea-83b6-4bb3-9bfe-63fa7e3cf607.png)

## 2. 문자열
 "**안녕하세요**"를 입력하여 추가 시에
![image](https://user-images.githubusercontent.com/73986381/166086517-27f72d72-e2d0-4a25-96ae-72cf35848213.png)

아래 사진과 같은 문구가 발생합니다. 에러 발생위치는 App.js 12번 라인이며, 이미지 첨부합니다.
문구 내용은 Uncaught TypeError: Cannot create property 'id' on string '안녕하세요' 
![image](https://user-images.githubusercontent.com/73986381/166086546-00c0b27f-a872-402e-960c-408d55692107.png)
![image](https://user-images.githubusercontent.com/73986381/166086593-bdff5a86-41f2-4402-b707-e76823580ba7.png)

id속성은 string으로 못 만든다고 에러가 발생하는 것 같습니다.
어떻게 에러 처리를 해야할 지 잘 모르겠습니다.

**Comment by fsoftwareengineer** [comment]
Add.js 에서 인풋필드와 item.title이 잘 연결되었는지 다시 확인 해주시기 바랍니다. 함수에 item오브젝트가 아니라 string이 넘어오는 이유는 input field에 값이 item.title이 아니라 item이 연결되었거나 인풋필드에 입력하는 핸들러가 item을 스트링으로 변경하는 것 같습니다.
